Change some messages to title caps

jessemchung opened the pull request:
Done by me and @in-verse

We ended up doing a couple to explore.  Do we need a demo video or anything?  I figured this was not necessary

Part of #2000.

kingthorin commented on zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties:
```diff
@@ -776,7 +776,7 @@ autoupdate.api.view.updatedAddons = Return a list of any add-ons that have been
 autoupdate.name = Auto-update Extension
 
-break.api.action.addHttpBreakpoint = Adds a custom HTTP breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.  
+break.api.action.addHttpBreakpoint = Adds a Custom HTTP Breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.  
```
This one could be left since its a sentence with a period. (It is used in a web UI component not in ZAP's GUI.)

jessemchung replied:
Right, makes sense, this change has been reversed.

kingthorin reviewed (commented):
Thanks. Welcome aboard.

jessemchung commented:
Oh I see, sorry about that.  We will fix it soon.  Thanks for looking it
over it for us.  We are both pretty new to this sort of thing.

On Fri, Sep 6, 2024 at 4:57 PM Rick M ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Thanks. Welcome aboard.
> ------------------------------
>
> In zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties
> <https://github.com/zaproxy/zaproxy/pull/8628#discussion_r1747707423>:
>
> > @@ -776,7 +776,7 @@ autoupdate.api.view.updatedAddons = Return a list of any add-ons that have been
>  autoupdate.desc = Allows ZAP to check for updates
>  autoupdate.name = Auto-update Extension
>
> -break.api.action.addHttpBreakpoint = Adds a custom HTTP breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.
> +break.api.action.addHttpBreakpoint = Adds a Custom HTTP Breakpoint. The string is the string to match. Location may be one of: url, request_header, request_body, response_header or response_body. Match may be: contains or regex. Inverse (match) may be true or false. Lastly, ignorecase (when matching the string) may be true or false.
>
> This one could be left since its a sentence with a period. (It is used in
> a web UI component not in ZAP's GUI.)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/zaproxy/zaproxy/pull/8628#pullrequestreview-2287146986>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATJFIITRJFO7LQKYL2BR643ZVIJJZAVCNFSM6AAAAABNZIX45SVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEOBXGE2DMOJYGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

kingthorin commented:
No problem. We all start somewhere.

jessemchung commented:
I have read the CLA Document and I hereby sign the CLA

jessemchung commented:
recheck

thc202 commented:
@In-Verse also needs to sign.

jessemchung commented:
@thc202 @kingthorin I'm really sorry, I think my friend got really busy with some personal stuff.  I decided to get a new pull request for just my things.  I can close this pull request as well if that is the correct thing to do.

This is the new pull request and I am really sorry for the trouble.
https://github.com/zaproxy/zaproxy/pull/8630

In-Verse commented:
I have read the CLA Document and I hereby sign the CLA

In-Verse commented:
This is my first time doing something like this. I have read the Contributor License Agreement. I believe my above comment is my signing and agreeing to the terms of the CLA. Please let me know if there's something more that I can do. 

Thank you for your patience!

kingthorin reviewed (commented):
Looks good to me. Do you think you can figure out how to do an interactive rebase and squash it all into one commit? (If that seem daunting we can do it at merge time.)

jessemchung commented:
@kingthorin 
So, I attempted to do the following

git rebase -i 914f41358069497268ff292dcaf578a4a9fb8e7f

With that commit being the commit before we added stuff
And I squashed all the commits except the first one I made.  It seems like that wasn't the right thing to do?

I have all the changes on one commit though, but not sure how to make it look better.
https://github.com/zaproxy/zaproxy/pull/8628/commits/cea6c84f235df5801cf9d944089825cb8f8ac7be

kingthorin commented:
It's okay. We'll take care of it.

kingthorin commented:
Forgot I had this saved, maybe it’ll help:

—-

Please do an interactive rebase and fix-up or squash into a single "atomic" commit: https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#what-we-zap-team-expect-from-you .

- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
- https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

jessemchung commented:
@kingthorin I think I got it into one commit.  Sorry for the trouble.  Turns out I needed to do an active rebase and than force the push.

kingthorin commented:
Thanks!

thc202 commented:
Was the removal of In-Verse on purpose?

kingthorin commented:
You can fix that by setting them as a co-author in the commit message

https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#

jessemchung commented:
@kingthorin @thc202 gotcha, I think we got it right.  Both our names are on the commit
